### DISCUSSION TITLE
How to find the internal service identifier of an iPhone device

### DISCUSSION THREAD

**Discussion by zmkUp** [doronz88/pymobiledevice3#798] [Q&A]
How to find the internal service identifier of an iPhone device?
like:
com.apple.instruments.server.services.networking
com.apple.instruments.server.services.deviceinfo

**Comment by jpstotz** [comment]
Are you asking how to get a list of available service identifiers of an iPhone?

**Comment by doronz88** [comment]
By reversing `lockdownd`

**Comment by doronz88** [comment]
The services you mentioned specifically are DVT channels you can look at the dvt shell example to just query them all
